cmd/api/src/model/appcfg/flag.go

```diff
@@ -114,15 +122,19 @@ type FeatureFlagSet map[string]FeatureFlag
 
 // FeatureFlagService defines a contract for fetching and setting feature flags.
 type FeatureFlagService interface {
+	GetFlagByKeyer
+
```

Comment:
Whats the advantage to pulling this out into a separate interface?

Reply:
Composability. We only care about the smaller interface in some uses, but the larger interface is older and used elsewhere